contrib/cirrus/setup.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -e
+
+UBUNTU_PACKAGES="
+    bzip2
+    cmake
+    coreutils
+    curl
+    gcc
+    git
+    openssl
+    pkg-config
+    python3
+    rsync
+    scons
+    vim
+    wget
+    yum-utils
+    zlib1g-dev
+    golang
+    go-md2man
+    libgpgme11-dev
+    libdevmapper-dev
+    libseccomp-dev
+    libglib2.0-dev
+    libostree-dev
+    bats
+    aufs-tools
+"
+
+FEDORA_PACKAGES="
+    bats
+    btrfs-progs-devel
+    bzip2
+    curl
+    device-mapper-devel
+    dnf-plugins-core
+    findutils
+    gcc
+    git
+    glib2-devel
+    go-md2man
+    golang
+    gpgme-devel
+    libassuan-devel
+    libseccomp-devel
+    ostree-devel
+    podman
+    python2
+    python3
```

Comment:
Do we really need both python2 and 3

Reply:
No idea at all TBH, this was mostly just copy-paste from somewhere else.

I'll add a comment, mentioning these lists needs to be tweaked.

Reply:
I don't think we need any python.